src/arch/util/machine-cma.C

```diff
@@ -3,6 +3,8 @@
 #endif
 #include <sys/uio.h>
 
+#define MAX_CMA_RW_COUNT 2147479552
```

Comment:
Maybe this should be the hexadecimal literal `0x7ffff000` for clarity.

Reply:
Done.